README.md

```diff
@@ -1,17 +1,9 @@
 # Code For America Disaster Response Hub
 
-## Installation
-
-This site is a static [Jekyll](https://jekyllrb.com/) site served on [GitHub Pages](https://pages.github.com/).
-
-See the Jekyll docs for [local installation](https://jekyllrb.com/docs/installation/)
-
-# Disaster Response Hub
-
 **Description**:  A quick response web hub for disaster response and sharing maps and data from [Shelter & POD API](https://github.com/hurricane-response/florence-api)
```

Comment:
Since this is the `response-theme` repo, we should describe it here as something like "a Jekyll theme for use in deploying quick-response web hubs for disaster response, using data from...". We need to call out the fact that it's a Jekyll theme, not a site.